samza-kafka/src/main/java/org/apache/samza/system/kafka/KafkaSystemConsumer.java

```diff
@@ -296,7 +296,7 @@ private static int compareOffsets(String offset1, String offset2) {
 
   @Override
   public String toString() {
-    return String.format("%s:%s", systemName, clientId);
+    return String.format("%s:%s@%s", systemName, clientId, Integer.toHexString(hashCode()));
```

Comment:
What is this for?

Reply:
used it for debugging (specific instance). Removed.